return of edit view

emilyville opened the pull request:
Add a history tab to the edit page to view previous episodes

Commit:
return of edit view

awong-dev commented on app/views/patients/_form_history.html.slim:
```diff
@@ -0,0 +1,49 @@
+      tr
+        th Previous Annual Evaluations
+      - for episode in @patient.annual_evaluations do
+        tr class="clickable-row" data-href=edit_polymorphic_url([@patient, episode])
```
You can use edit_annual_evluations_url, etc. all over now since you know the type. can you get rid of all the polymorphic_url bits?

awong-dev committed:
Remove polymoprhic_url

Since we know how to not be polymorphic now.

awong-dev committed:
Sort the dates in history display.

Also fix the "no episode of care" display to be
episode type specific.

awong-dev committed:
Remove more reflection.

Also, it seems pointless to list the episode of care type if it's
already in the box with the name.

awong-dev committed:
Factor out forms. Fix Acute Rehab edit.

awong-dev commented:
@emilyville Your changes LGTM

Back at you with mine.

emilyville commented:
:+1:
